src/components/useRealtimeAPI.tsx

```
@@ -0,0 +1,264 @@
+      const jsonData = JSON.parse(event.data)
+      console.log('Received message:', jsonData.type)
+
+      switch (jsonData.type) {
+        case 'error':
+          console.log('エラーデータを受信しました', jsonData)
+        case 'conversation.item.created':
+          console.log('コンテキストデータを受信しました', jsonData)
+        case 'response.audio.delta':
+          console.log('オーディオデータを受信しました', jsonData)
+          if (jsonData.delta) {
+            const arrayBuffer = base64ToArrayBuffer(jsonData.delta)
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)